server/app.js

```diff
@@ -1,32 +1,31 @@
 import express from 'express';
```

Comment:
In general.
- Config absolute path
- Omit .js suffix from imported files
- Make an URL instance separate by feature for the CRUD (like FE side)